homeassistant/components/vallox/strings.json

```diff
@@ -3,19 +3,22 @@
     "step": {
       "user": {
         "title": "Vallox",
-        "description": "Set up the Vallox integration. If you have problems with configuration go to https://www.home-assistant.io/integrations/vallox.",
+        "description": "Set up the Vallox integration. If you have problems with configuration go to {integration_docs_url}.",
         "data": {
-          "host": "[%key:common::config_flow::data::host%]",
-          "name": "[%key:common::config_flow::data::name%]"
+          "host": "[%key:common::config_flow::data::host%]"
         }
       }
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_host": "[%key:common::config_flow::error::invalid_host%]",
+      "unknown": "Unknown error"
```

Comment:
Please use this common reference:
https://github.com/home-assistant/core/blob/e3c1a3567ee76aaf56aa1d61bbed6e2e6354a155/homeassistant/strings.json#L59

Reply:
`[%key:common::config_flow::error::unknown%]`

Reply:
Fixed. 👍